### PR TITLE
Cleanup groovy

### PR DESCRIPTION
**Pull request by jsoref**
This is for discussion.

Notes:
* Commits were created by IntelliJ
* Individual commits are mostly self contained
* In general, most commits should not change program behavior at all

### Proposed changelog entries

* `Internal:` Internal Groovy code cleanup

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

**Review by jvz — APPROVED** [review]

File: war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy [hunk 1]
```diff
@@ -33,15 +33,10 @@ import hudson.security.ChainedServletFilter
 import hudson.security.UnwrapSecurityExceptionFilter
 import hudson.security.HudsonAuthenticationEntryPoint
 import jenkins.security.BasicHeaderProcessor
-import org.acegisecurity.providers.anonymous.AnonymousProcessingFilter
 import jenkins.security.ExceptionTranslationFilter
-import org.acegisecurity.ui.basicauth.BasicProcessingFilter
+import org.acegisecurity.providers.anonymous.AnonymousProcessingFilter
 import org.acegisecurity.ui.basicauth.BasicProcessingFilterEntryPoint
 import org.acegisecurity.ui.rememberme.RememberMeProcessingFilter
-import hudson.security.HttpSessionContextIntegrationFilter2
```
Comment (oleg-nenashev) [inline war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy]: Due to whatever reason, it causes the ATH failure:

```
Jenkins error detected:
groovy.lang.MissingPropertyException: No such property: HttpSessionContextIntegrationFilter2 for class: hudson.util.spring.BeanBuilder
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.unwrap(ScriptBytecodeAdapter.java:53)
at org.codehaus.groovy.runtime.callsite.PogoGetPropertySite.getProperty(PogoGetPropertySite.java:52)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGroovyObjectGetProperty(AbstractCallSite.java:307)
at Script1$_run_closure1.doCall(Script1.groovy:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
```

Comment (jsoref) [reply war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy]: My guess is this is https://github.com/jenkinsci/jenkins/blob/22aa2e6e766074d11249893e3f35e0b99e20d3d0/war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy#L66

File: war/src/main/webapp/WEB-INF/security/AbstractPasswordBasedSecurityRealm.groovy [hunk 2]
```diff
@@ -37,7 +37,7 @@ authenticationManager(ProviderManager) {
 
         // these providers apply everywhere
         bean(RememberMeAuthenticationProvider) {
-            key = Jenkins.getInstance().getSecretKey();
+            key = Jenkins.get().getSecretKey()
```
Comment (Wadeck) [inline war/src/main/webapp/WEB-INF/security/AbstractPasswordBasedSecurityRealm.groovy]: For future reviewers => only real change

File: war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy [hunk 3]
```diff
@@ -33,15 +33,13 @@ import hudson.security.ChainedServletFilter
...
 import hudson.security.HudsonAuthenticationEntryPoint
 import jenkins.security.BasicHeaderProcessor
-import org.acegisecurity.providers.anonymous.AnonymousProcessingFilter
 import jenkins.security.ExceptionTranslationFilter
-import org.acegisecurity.ui.basicauth.BasicProcessingFilter
+import org.acegisecurity.providers.anonymous.AnonymousProcessingFilter
 import org.acegisecurity.ui.basicauth.BasicProcessingFilterEntryPoint
 import org.acegisecurity.ui.rememberme.RememberMeProcessingFilter
+
+/* required by a bean/hudson.util.spring.BeanBuilder: */
```
Comment (Wadeck) [inline war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy]: As this is put directly in the script, I do not think this is necessary (especially a comment inside the import, the probability to be read is very low)

If you do not mind removing this, I would be happy 🥇

**Review by Wadeck — APPROVED** [review]
:bee:

Thank you for the cleanup, always burning eyes to see Groovy script with semicolon ;)

File: war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy (hunk 3 above)
Comment (jsoref) [reply war/src/main/webapp/WEB-INF/security/SecurityFilters.groovy]: Removed

**Review by oleg-nenashev — APPROVED** [review]
LGTM. TBH I see no value in keeping groovy web UI rendering, but it is unrelated to this PR

**Pull request merged**